src/consensus/pbft/libbyz/Replica.cpp

```diff
@@ -1045,6 +1064,17 @@ void Replica::register_global_commit(global_commit_handler_cb cb, void* ctx)
   global_commit_ctx = ctx;
 }
 
+template <class T>
+std::unique_ptr<T> Replica::create_message(
+  const uint8_t* message_data, size_t data_size)
+{
+  Message* m = new Message(Max_message_size);
```

Comment:
you do not to alloc max_message_size as you know the type you are converting to.